docs-src/references/dynamic-configuration.md

```diff
@@ -67,8 +67,8 @@ Not defining constraints on a dynamic configuration key sets the values globally
 - To set global values for the configuration key with no constraints, use:
 
   ```yaml
-  history.rps: # Rate limit on the number of requests made per second to each History service host.
-    - value: 5000 # The default value for this is 3000.
+  frontend.globalNamespaceRPS: # Rate limit on the number of requests made per second to each History service host.
+    - value: 5000 # The default value for this is 2400.
```

Comment:
that key has no default

Reply:
Yeah; I put that default there because if it is not set, I assumed the `frontend.namespacerps` limit would apply (https://github.com/temporalio/temporal/blob/a51abcfffb28cda76049588a81f3518d0a3e4651/common/dynamicconfig/constants.go#L232-L235).
So if there is a global namespace configured, but this value is not set, will it allow unlimited requests?

Reply:
Yes, basically if frontend.globalNamespaceRPS isn't set then frontend.namespaceRPS will take effect. But I think it's misleading to say that the default for frontend.globalNamespaceRPS is 2400: the two keys have different meanings. The 2400 is per instance, the value for frontend.globalNamespaceRPS is total for all instances

Reply:
got it; I've removed the comment. But this is an important detail that I think is best covered in the Cluster deployment and monitoring. Will keep it in mind.